.gitignore

```diff
@@ -1,2 +1,5 @@
 vendor
 devbin/*
+go1.22.4.linux-amd64.tar.gz
+yezzey.log
+examples/yproxy.yaml
```

Comment:
examples/yproxy.yaml definitely shouldn't persist here, as this file is committed into repo.

Reply:
also go1.22.4.linux-amd64.tar.gz is not needed here